app/views/users/show.html.erb

```diff
@@ -1,35 +1,54 @@
-<div>
-  <h1>Users Details page</h1>
-  <h1> Name: <%= @user.name %> </h1>
-  <h4>
-    Total post: <%= @user.post_counter %>
-  </h4>
-  <img width="200px" src=<%=@user.photo %> alt=<%= @user.name %> />
+<div class="flex  w-screen items-center justify-center mx-9 px-4 py-4 sm:px-6 lg:px-8">
 
-  <h3>Bio</h3>
-  <p>
-    <%= @user.bio %>
-  </p>
-  <hr>
-  <h1>Post</h1>
-  <hr>
-  <%@posts.each_with_index do |post, index| %>
-  <div>
-    <li>
-      <%=index +1 %> - <%= post.title %>
-      <p>
-        Post Content: <%= post.text %>
+  <div class=" w-2/4">
+    <h1 class="m-4 text-4xl font-extrabold tracking-tight leading-none text-gray-900 md:text-5xl lg:text-6xl">
+      Users Details page
+    </h1>
+    <div
+      class="flex flex-col items-center bg-white rounded-lg border shadow-md md:flex-row  hover:bg-gray-100 dark:border-gray-700 dark:bg-gray-800 dark:hover:bg-gray-700">
+      <img class="object-cover w-full h-96 rounded-t-lg md:h-auto md:w-48 md:rounded-none md:rounded-l-lg"
+        src="<%=@user.photo %>" alt="">
+      <div class="flex flex-col justify-between p-4 leading-normal">
+        <h5 class="mb-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white">Name: <b><%=@user.name %></h5>
+        <p class="mb-3 font-normal text-gray-700 dark:text-gray-400">Bio: <%= @user.bio %></p>
+        <p class="mb-3 font-normal text-gray-700 dark:text-gray-400"> Number of Post: <%= @user.post_counter %></p>
+      </div>
+    </div>
 
-      </p>
+    <h1 class="m-4 text-3xl font-extrabold tracking-tight leading-none text-gray-900">
+    Posts</h1>
+    <%@posts.each_with_index do |post, index| %>
```

Comment:
- [ ] Awesome job here. But, you are required to use the `recent_posts` method in your user model to display the 3 most recent posts. 👍

Reply:
This is not part of the current requirement. It was made for because of styling. Why are you looking for errors?